Envfile

```diff
@@ -164,4 +164,21 @@ variable :SMARTY_STREETS_WEB_KEY, :String, default: "Optional"
 
 group :production do
   variable :SECRET_KEY_BASE, :String
+  
+  # Timber
+  variable :SEND_LOGS_TO_TIMBER, :String  # Defaults to true in code
```

Comment:
The description should probably say "Defaults to true in production", since it defaults to false in development mode